Formula/r/runitor.rb

```diff
@@ -0,0 +1,22 @@
+class Runitor < Formula
+  desc "Command runner with healthchecks.io integration"
+  homepage "https://github.com/bdd/runitor"
+  url "https://github.com/bdd/runitor/archive/refs/tags/v1.3.0-build.4.tar.gz"
```

Comment:
raised an upstream issue about the release scheme, https://github.com/bdd/runitor/issues/178

Reply:
Amended to not use the .build-4 version as suggested by upstream

Reply:
make sense 👍